### DISCUSSION TITLE
clang 16 unsafe-buffer-usage

### DISCUSSION THREAD

**Discussion by idkfaiddqd** [NVIDIA-Omniverse/PhysX#264] [General]
Hello!

I'm trying to build latest Physx with ubuntu and clang 16 and getting a lot of  errors 

`PhysX/physx/source/foundation/FdString.cpp:122:21: error: 'str' is an unsafe pointer used for buffer access [-Werror,-Wunsafe-buffer-usage]`

What is the right way to tune the build configuration to avoid erors?

Thanks!

**Comment by preist-nvidia** [comment]
We only test and support certain specific compiler versions, please refer to: https://github.com/NVIDIA-Omniverse/PhysX/blob/main/physx/documentation/platformreadme/linux/README_LINUX.md

**Comment by idkfaiddqd** [comment]
Great thanks!
